capstone/src/main/java/com/google/univiz/CollegeData.java

```diff
@@ -0,0 +1,50 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * CollegeData is a class representing a single university/college/institution. These parameters use
+ * data parsed from the CollegeScorecard API. This data has been converted from the ScorecardData
```

Comment:
i wouldn't mention scorecard here since it's meant to be agnostic of what implementation is providing the data. These sort of comments would be great in the implementation class that implements this interface.